src/main/java/mate/jdbc/dao/ManufacturerDaoImpl.java

```diff
@@ -0,0 +1,122 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exeption.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class ManufacturerDaoImpl implements ManufacturerDao {
+    private static final int PARAMETER_INDEX_NAME = 1;
+    private static final int PARAMETER_INDEX_COUNTRY = 2;
+    private static final int PARAMETER_INDEX_ID = 3;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        String insertFormatRequest = "INSERT INTO manufacturers(name, country) values(?, ?);";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createFormatStatement = connection
+                        .prepareStatement(insertFormatRequest, Statement.RETURN_GENERATED_KEYS)) {
+            createFormatStatement.setString(PARAMETER_INDEX_NAME, manufacturer.getName());
+            createFormatStatement.setString(PARAMETER_INDEX_COUNTRY, manufacturer.getCountry());
+            createFormatStatement.executeUpdate();
+            ResultSet generatedKeys = createFormatStatement.getGeneratedKeys();
+            if (generatedKeys.next()) {
+                Long id = generatedKeys.getObject(PARAMETER_INDEX_NAME, Long.class);
+                manufacturer.setId(id);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException("Can't insert manufacturer to DB", e);
```

Comment:
add more info to exception, as a variant 
```suggestion
            throw new DataProcessingException("Can't insert manufacturer = " + manufacturer +  "to DB", e);
```